api/jreleaser-model-api/src/main/java/org/jreleaser/version/CalVer.java

```diff
@@ -70,8 +70,8 @@ public class CalVer implements Version<CalVer> {
 
     static {
         PATTERNS.put(T_YEAR_LONG, "([2-9][0-9]{3})");
-        PATTERNS.put(T_YEAR_SHORT, "([1-9]|[1-9][0-9]|[1-9][0-9]{2})");
-        PATTERNS.put(T_YEAR_ZERO, "(0[1-9]|[1-9][0-9]|[1-9][0-9]{2})");
+        PATTERNS.put(T_YEAR_SHORT, "([0-9]|[1-9][0-9]|[1-9][0-9]{2})");
```

Comment:
Why does T_YEAR_SHORT need an update?

Reply:
Here `"0"` is passed, but it won't match `"([1-9]|[1-9][0-9]|[1-9][0-9]{2})"` pattern:
https://github.com/jreleaser/jreleaser/blob/7e909958b12823fe5cd9427cfde889fa496ee21c/api/jreleaser-model-api/src/main/java/org/jreleaser/version/CalVer.java#L467